cmd/opencensusd/exporters.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+
+	datadog "github.com/DataDog/opencensus-go-exporter-datadog"
+	openzipkin "github.com/openzipkin/zipkin-go"
+	"github.com/openzipkin/zipkin-go/reporter/http"
+	"go.opencensus.io/exporter/stackdriver"
+	"go.opencensus.io/exporter/zipkin"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/trace"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var (
+	statsExporters []view.Exporter
+	traceExporters []trace.Exporter
+	flushers       []func()
+)
+
+func exportViewData(vd *view.Data) {
+	// TODO(jbd): Implement.
+}
+
+func exportSpanData(sd *trace.SpanData) {
+	for _, exporter := range traceExporters {
+		exporter.ExportSpan(sd)
+	}
+}
+
+func flush() {
+	for _, fn := range flushers {
+		fn()
+	}
+}
+
+type config struct {
+	Datadog struct {
+		// Namespace specifies the namespaces to which metric keys are appended.
+		Namespace string `yaml:"namespace_addr,omitempty"`
```

Comment:
I think we should use camelCase for yaml attributes, since this is more consistent with Kubernetes config.

Reply:
Done.